hotpot_km/tests/test_initialize.py

```diff
@@ -0,0 +1,130 @@
+
+import asyncio
+from contextlib import asynccontextmanager
+import mock
```

Comment:
```suggestion
from unittest import mock
```
available from 3.3 https://docs.python.org/3.6/library/unittest.mock.html

Reply:
Ah, I got them reversed.